js/client/modules/@arangodb/testutils/instance.js

```diff
@@ -514,6 +514,14 @@ class instance {
   // //////////////////////////////////////////////////////////////////////////////
 
   readAssertLogLines () {
+    let size = fs.size(this.logFile);
+    if (size > 500000) {
+      // File bigger 500k? this needs to be a bug in the tests.
+      let err=`ERROR: ${this.logFile} is bigger than 500kB! - %{size} Bytes!`;
+      this.assertLines.push(err);
+      print(RED + err + RESET);
+      return;
```

Comment:
I don't think it should be like this. There might be a need to enable TRACE or DEBUG and turn this check off. Moreover, 501kB doesn't sound like a huge log.